bedrock/firefox/templates/firefox/privacy/index.html

```diff
@@ -88,65 +62,61 @@ <h3 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-we-do-the-hard
           {{ ftl('firefox-privacy-hub-data-security-is-complicated') }}
         {% endif %}
       </p>
-    {% endcall %}
+    </div>
+  </div>
+
+  <div>
+    <h2>{{ ftl('firefox-privacy-hub-no-secrets') }}</h2>
+    <h4 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-youll-always-know-where-you') }}</h4>
+    <p>
+      {% if ftl_has_messages('firefox-privacy-hub-theres-no-hidden-agenda-here-v2') %}
+        {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here-v2', privacy=url('privacy.notices.firefox')) }}
+      {% else %}
+        {{ ftl('firefox-privacy-hub-theres-no-hidden-agenda-here',
+          meetings='https://wiki.mozilla.org/WeeklyUpdates?utm_source=' ~ _entrypoint ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign ~ '&entrypoint=' ~ _entrypoint,
+          privacy=url('privacy.notices.firefox')) }}
+      {% endif %}
+    </p>
+  </div>
+</div>
 
-    {% call split(
-      image=resp_img('img/firefox/privacy/promise/secrets.jpg', srcset={ 'img/firefox/privacy/promise/secrets-high-res.jpg': '2x' }, optional_attributes={ 'class': 'mzp-c-split-media-asset'}),
-      block_class='privacy-promise-feature mzp-l-split-reversed'
-    ) %}
-      <h2>{{ ftl('firefox-privacy-hub-no-secrets') }}</h2>
-      <h3 class="privacy-promise-sub-title">{{ ftl('firefox-privacy-hub-youll-always-know-where-you') }}</h3>
+<div class="privacy-promise-learn-more">
```

Comment:
```suggestion
<div class="mzp-t-background-secondary">
```

**Change:** There's a standard class to give something the grey background now, so we can remove the custom code _and_ keep the background.